### PR TITLE
docs(i18n-id): translate jsx, jsxFactory & jsxFragmentFactory into Indonesia

### PR DESCRIPTION
**Pull request by pandurijal**
## Overview
This is part of #938

## Changes
Translate the following files

 - [x] jsx.md
  - [x] jsxFactory.md
  - [x] jsxFragmentFactory.md

### CODE REVIEW

**Comment by jefrydco** [comment]
Mas @pandurijal minta tolong `Contributor License Agreement` di atas di _sign_ dulu ya

**Comment by pandurijal** [comment]
Okey sudah mas @jefrydco

File: packages/tsconfig-reference/copy/id/options/jsxFactory.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+display: "JSX Factory"
+oneline: "Mengatur fungsi yang dihasilkan oleh JSX"
+---
+
+Mengubah fungsi yang dipanggil pada file `.js` ketika melakukan kompilasi elemen-elemen JSX.
+Perubahan yang paling umum adalah dengan menggunakan `"h"` atau `"preact.h"` dibandingkan penggunaan default `"React.createElement"` jika menggunakan `preact`.
```
Comment (mazipan) [inline packages/tsconfig-reference/copy/id/options/jsxFactory.md]: default = bawaan

**Review by mazipan — CHANGES_REQUESTED** [review]

File: packages/tsconfig-reference/copy/id/options/jsxFactory.md (hunk 1 above)
Comment (pandurijal) [reply packages/tsconfig-reference/copy/id/options/jsxFactory.md]: done @irfan-maulana-tkp

**Review by mandaputtra — APPROVED** [review]

**Review by mazipan — APPROVED** [review]
Looks good 👍

**Comment by mazipan** [comment]
LGTM

**Pull request merged**